msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -209,6 +209,16 @@ To preview your app running in Office on the web:
 
     :::image type="content" source="images/office-web-more-apps.png" alt-text="Click on the 'More apps' option on the side bar of office.com to see your installed personal tabs":::
 
+### Office for Android
+
+To view your app running in Office for Android:
+
+1. Launch the Office app and sign in using your dev tenant account.
+1. Select the **Apps** icon. Your sideloaded app appears among installed apps.
```

Comment:
Do we need to put a note about ensuring, user should join beta and app version should be 16.0.15629.20000 and above?

Reply:
added a note to the new commit